packages/localstorage-core/README.md

```diff
@@ -8,7 +8,7 @@
 ## Usage
 
 ```js
-import AvLocalStorarge from '@availity/localstorage-core`;
+import AvLocalStorage from '@availity/localstorage-core`;
```

Comment:
Could have also fixed that backtick 😉

Reply:
https://github.com/Availity/sdk-js/commit/1a893b5f2ea33b21b89a61a628b428a2c058e9ac 👀